README.md

```diff
@@ -54,6 +54,9 @@ manager:
 # macOS or Linux
 brew tap charmbracelet/tap && brew install charmbracelet/tap/soft-serve
 
+## Windows (with Winget)
+winget install soft-serve
```

Comment:
```suggestion
# Windows (with Winget)
winget install charmbracelet.soft-serve
```

Reply:
@aymanbagabas Thanks for the suggestion, will apply it. I am fine with using the full identifier (feel free to checkout the [above comment](https://github.com/charmbracelet/soft-serve/pull/304#discussion_r1423395357) explaining Moniker)